prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -2125,8 +2121,10 @@ subroutine getSccHamiltonian(H0, over, nNeighbourSK, neighbourList, species, orb
 
     nAtom = size(orb%nOrbAtom)
 
-    ham(:,:) = 0.0_dp
-    ham(:,1) = h0
+    if (.not. isREKS) then
+      ham(:,:) = 0.0_dp
```

Comment:
The variable ham is intent out, so should probably be set to zero before this if test. Internally the add_shift() routine treats it as intent(inout) and just adds to its value, so it might not be initialized for REKS.

Reply:
Done with commit b2264d1.